src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -383,6 +399,20 @@ public String getLocation() {
       return location;
     }
 
+    @Nullable
+    public String getContentType() {
```

Comment:
what happens if someone uses content type as `text/plain`, and format as `avro`? Do we have any validations that would prevent that error configuration during deployment

Reply:
When someone uses content type as text/plain and format as avro, the pipeline runs successfully since we don't have any validation that would prevent any error during the deployment.

Reply:
PR Updated with necessary validations and unit tests.